pages/data.md

```diff
@@ -1,12 +1,12 @@
 # Butterfly Hybrid Detection: Data
 
-This challenge uses data compiled from various Zenodo records of the Butterfly Genetics Group. [More Info Coming]
+This challenge uses data compiled from various Zenodo records of the Butterfly Genetics Group.
 
 ## Instructions to Download Training Data
 
 First, install the downloader in your virtual environment:
 ```bash
-pip install git+https://github.com/Imageomics/cautious-robot.git@v0.1.1-alpha
+pip install git+https://github.com/Imageomics/cautious-robot.git@v0.2.0-alpha
```

Comment:
`sum-buddy` is still on alpha release; would it be reasonable to just change to install from the repo (no release)?

Reply:
Yes in principle, with the caveat being that then someone who runs this 3 months from now may get something different from someone who runs it now. But that could be intended – for example, if there are improvements and/or bug fixes on the main branch by then, we would probably want someone to get these rather than a version that at that point won't have them.

I guess if we have people install from the main branch, we essentially commit ourselves to keeping that branch backwards compatible and tested for the duration of this competition. If we don't want to make that commitment, we should designate perhaps a special branch, to which we would keep migrating fixes from the main branch for the duration of the competition.

Reply:
And as an aside, that a dependency of a software is alpha by no means has to imply that the software itself becomes "tainted" as alpha then too. If your software has a dependency that is alpha but your software otherwise is not, you essentially take responsibility for having checked that it works to the extent you need it to work. (The dependency may be alpha for reasons that don't matter to your software.)

Reply:
There is one more large-ish change we were discussing ([here](https://github.com/Imageomics/cautious-robot/issues/15#issuecomment-2273710520)). I think we have pretty good testing in place, so we could update and do a release. However, I think that just having the newest version would likely be best; I don't intend to change the input in a way that wouldn't be backwards compatible.